app/main.py

```diff
@@ -31,12 +31,12 @@
 import base64
 import requests
 
+from cryptography.hazmat.primitives import serialization
```

Comment:
For now this looks good.  The comments from the developers of the hazmat are:

![image](https://github.com/Michael-Sinz/acs-keyvault-agent/assets/36865706/bc39ccaa-4cad-49a1-841a-9de722aeb230)

Reply:
Just glad it does not have sharks with lasers :-)

Reply:
Yeah, I hesitated because I've never fought a dinosaur with a laser gun before.